src/main/java/com/iota/iri/Iota.java

```diff
@@ -37,21 +38,20 @@
 
 /**
  *
- * The main class of IRI. This will propagate transactions into and throughout the network.
- * This data is stored as a {@link Tangle}, a form of a Directed acyclic graph.
- * All incoming data will be stored in one or more implementations of {@link PersistenceProvider}.
+ * The main class of IRI. This will propagate transactions into and throughout the network. This data is stored as a
+ * {@link Tangle}, a form of a Directed acyclic graph. All incoming data will be stored in one or more implementations
+ * of {@link PersistenceProvider}.
  *
  * <p>
- *     During initialization, all the Providers can be set to rescan or revalidate their transactions.
- *     After initialization, an asynchronous process has started which will process inbound and outbound transactions.
- *     Each full node should be peered with 7-9 other full nodes (neighbors) to function optimally.
+ * During initialization, all the Providers can be set to rescan or revalidate their transactions. After initialization,
+ * an asynchronous process has started which will process inbound and outbound transactions. Each full node should be
+ * peered with 7-9 other full nodes (neighbors) to function optimally.
```

Comment:
3-5 other full nodes in stead of 7-9 ?

Reply:
Changed to 3-5

Reply:
Still seeing 
 `*     Each full node should be peered with 7-9 other full nodes (neighbors) to function optimally.	 `
?

Reply:
I am seeing it correctly at my IDE.
Maybe you are looking at the outdated snippet?

Reply:
Yep, missed the Outdated label on top, sorry about that.